src/SprykerEco/Zed/ComputopApi/Business/Request/PostPlace/AuthorizationRequest.php

```diff
@@ -7,6 +7,9 @@
 
 namespace SprykerEco\Zed\ComputopApi\Business\Request\PostPlace;
 
+/**
+ * @method \Generated\Shared\Transfer\ComputopApiAuthorizeResponseTransfer request(\Generated\Shared\Transfer\OrderTransfer $orderTransfer, \Generated\Shared\Transfer\ComputopApiHeaderPaymentTransfer $computopApiHeaderPayment)
```

Comment:
Why do we need this? We. already have this method with all needed hints in abstraction

Reply:
It was added because every of this requests return different response transfer it fixes some phpstan issues and add phpstorm autocompletion.